### ISSUE TITLE
Full text wrapped in square brackets

### ISSUE THREAD

**Issue by smythp** [JonathanReeve/corpus-db#25]
When getting a full text, like in this example:

    http://corpus-db.org/api/id/108/fulltext

the result is wrapped in square brackets. That means that when loaded into JSON it's a list with 1 item in it (the dictionary) rather than just a dictionary. Think it would be better without the brackets so it wouldn't need to be parsed out.

**Comment by JonathanReeve** [comment]
Good point. I just use one function for getting full text of many texts, and for getting full text of a single text. But a single text shouldn't need to be a list of length one. I'll make a function that returns single texts.
